crates/cli/src/utils/cmd.rs

```diff
@@ -140,6 +140,7 @@ pub fn eta_key(state: &indicatif::ProgressState, f: &mut dyn Write) {
     write!(f, "{:.1}s", state.eta().as_secs_f64()).unwrap()
 }
 
+#[allow(clippy::literal_string_with_formatting_args)]
```

Comment:
can you allow it globally and add todo linking to https://github.com/rust-lang/rust-clippy/issues/13885